core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiment-options/directives/dot-experiment-option-content.directive.ts

```diff
@@ -0,0 +1,11 @@
+import { Directive, TemplateRef } from '@angular/core';
+
+/**
+ * Get the content TemplateRef of the Option
+ */
+@Directive({
+    selector: '[dotOptionContent]'
+})
+export class DotExperimentOptionContentDirective {
+    constructor(public templateRef: TemplateRef<unknown>) {}
+}
```

Comment:
What is the goal of this `directive`? I know this directive is being used in an `ng-template`. However, I don't know where the `templateRef` is being used.